src/auth/AclProxyAuth.cc

```diff
@@ -187,4 +189,3 @@ ACLProxyAuth::matchProxyAuth(ACLChecklist *cl)
     checklist->auth_user_request = NULL;
     return result;
 }
-
```

Comment:
Please avoid out-of-scope whitespace changes.

Reply:
Forgive me for inattention with out-of-scope whitespace.